ecloud/resource_loadbalancer_network.go

```diff
@@ -0,0 +1,156 @@
+package ecloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	ecloudservice "github.com/ukfast/sdk-go/pkg/service/ecloud"
+)
+
+func resourceLoadBalancerNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLoadBalancerNetworkCreate,
+		Read:   resourceLoadBalancerNetworkRead,
+		Update: resourceLoadBalancerNetworkUpdate,
+		Delete: resourceLoadBalancerNetworkDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"network_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Could do with `ForceNew: true` for this property

Reply:
added